package.json

```diff
@@ -2,56 +2,67 @@
-    "type": "library",
-    "template": {
-      "exclude": ["jest.config.js", ".github/workflows/package.yml"],
-      "documentation": "docusaurus"
-    }
-  },
-  "engines": {
-    "node": ">=16"
+  "license": "MIT",
+  "author": "SkyLeague Technologies B.V. <technologies@skyleague.io> (https://skyleague.io)",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js",
+    "./package.json": "./package.json",
+    "./*.js": "./dist/*.js"
   },
-  "files": ["dist", "package.json"],
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist",
+    "package.json"
+  ],
   "scripts": {
-    "build": "npx ts-node build.config.ts",
-    "build:clean": "npx tsc --build --clean && rm .main.js && rm .main.js.map",
+    "build": "npx ts-node --esm build.config.ts",
+    "build:clean": "npx tsc --build --clean && rm -rf dist",
     "build:docs": "(cd docs; npm install; npm run build)",
     "check:cost": "npx cost-of-modules --no-install --include-dev",
-    "check:coverage": "npx jest --collectCoverage=true",
+    "check:coverage": "NODE_OPTIONS=\"$NODE_OPTIONS --experimental-vm-modules\" npx jest --collectCoverage=true",
     "check:full": "npm run lint && npm run check:types && npm run check:coverage && npm run build && npm run check:project",
     "check:project": "npx node-standards lint",
     "check:types": "npx tsc -p tsconfig.json",
-    "format": "npx prettier \"**/*.{ts,js,json,yml,yaml,md}\" --ignore-path .gitignore --write",
-    "lint": "npx eslint \"{src,test,typing}/**/*.ts\" --no-eslintrc --cache -c .eslintrc.js --ignore-path .gitignore",
+    "format": "npx prettier \"**/*.{ts,js,json,yml,yaml,md}\" --write",
+    "lint": "npx eslint \"./{src,test,typings}/**/*.ts\" --cache --no-eslintrc --config .eslintrc.cjs",
     "lint:fix": "npm run lint -- --fix",
     "lint:full": "bash -c \"FULL_LINT=true npm run lint\"",
     "package": "rm -rf .dist && npm run build",
     "prepare": "npx husky install || true",
-    "test": "npx jest"
+    "test": "NODE_OPTIONS=\"$NODE_OPTIONS --experimental-vm-modules\" npx jest"
   },
-  "publishConfig": {
-    "registry": "https://registry.npmjs.org",
-    "access": "public"
+  "dependencies": {
+    "fast-deep-equal": "^3.1.3",
+    "node-object-hash": "^2.3.10",
+    "tslib": "^2.5.0"
   },
   "devDependencies": {
-    "@skyleague/node-standards": "^1.12.2",
+    "@skyleague/node-standards": "1.22.0-beta-2172eaee.1",
     "date-fns": "^2.29.3",
-    "jest-extended": "^3.2.0",
-    "typescript": "^4.9.3"
+    "esbuild": "^0.17.14",
+    "jest-extended": "^3.2.4",
+    "typescript": "^5.0.2"
   },
-  "dependencies": {
-    "fast-deep-equal": "^3.1.3",
-    "node-object-hash": "^2.3.10",
-    "tslib": "^2.4.1"
+  "engines": {
+    "node": ">=18"
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org"
+  },
+  "node-standards": {
+    "extends": [
+      "library",
+      "docusaurus"
+    ],
+    "ignorePatterns": [
+      "jest.config.cjs"
```

Comment:
why is the ignore needed?

Reply:
we have some different "fakeTimer" settings compared to the defaults